indigo_app/templates/indigo_api/_task_detail.html

```diff
@@ -100,7 +100,7 @@ <h5>
                   <div class="sheet-outer"
                        {% if anchor.selectors or not anchor.is_toc_element %}data-highlight="{{ anchor.target|jsonify|escape }}"{% endif %}>
                     <div class="sheet-inner is-fragment">
-                      <div class="akoma-ntoso country-{{ anchor.document.country }}">{{ anchor.toc_element_html|default:anchor.element_html|safe }}</div>
+                      <la-akoma-ntoso frbr-country="{{ anchor.document.country }}">{{ anchor.toc_element_html|default:anchor.element_html|safe }}</la-akoma-ntoso>
```

Comment:
not sure if I have access to the full frbr-uri expression, so I just went with country and I couldn't find the page in the front end either. I will happily change it to frbri-uri if we have access to that value in `anchor.document`

Reply:
Use `anchor.document.expression_frbr_uri`.

To test: in a document, add a comment. In the comment, click Create Task. Then open that task.

Reply:
done